components/add_users_to_team/add_users_to_team.jsx

```diff
@@ -45,6 +46,8 @@ export default class AddUsersToTeam extends React.Component {
             addError: null,
             loadingUsers: true,
         };
+
+        this.renderOption = this.renderOption.bind(this);
```

Comment:
Thanks for the notice. I forgot about this benefit of arrow functions when I was writing this code. Nevertheless, this line is now unnecessary since I use just the `AddIcon` component inside `renderOption` instead of `this.context.intl.formatMessage` and I don't need to have access to `this`. I deleted this line in the last update of this PR.